fighters/mariod/src/acmd/aerials.rs

```diff
@@ -50,6 +50,21 @@ unsafe fn mariod_attack_air_n_effect(fighter: &mut L2CAgentBase) {
     }
 }
 
+#[acmd_script( agent = "mariod", script = "sound_attackairn" , category = ACMD_SOUND , low_priority)]
+unsafe fn sound_attackairn(fighter: &mut L2CAgentBase) {
+    let lua_state = fighter.lua_state_agent;
+    let boma = fighter.boma();
```

Comment:
Are not keeping the vanilla "se_common_punch_kick_swing_s" sfx call on frame 3?

Reply:
I figured it'd make more sense to call just some electric sfx for the sweetspot so I didn't bother dumping/getting the vanilla sound script first, but if you think both should be there then that's something I can do

Reply:
> I figured it'd make more sense to call just some electric sfx for the sweetspot so I didn't bother dumping/getting the vanilla sound script first, but if you think both should be there then that's something I can do

Ye it comes from the initial kick (frame 3). Just sells it a bit more

Reply:
Added